src/extension.ts

```diff
@@ -249,6 +250,7 @@ export async function activate(context: vscode.ExtensionContext) {
                 }
             });
             jobsProvider.refresh();
+            Profiles.getInstance().refresh();
```

Comment:
I didn't consider they would as they operate on completely separate areas. The Jobs would simply get the initial children of the root via a zosmf call whilst the Profiles refresh will cause the CLIProfilleManager to do it's thing again. I may be wrong and if I thought they crossed over at all I would have managed them differently. One thing I haven't really considered is background threads. That's because I come from a Java background rather than JavaScript and haven't found those tricks yet